k8s/helm-chart/dynatrace-gcp-monitor/Chart.yaml

```
@@ -13,7 +13,7 @@
 #     limitations under the License.
 
 apiVersion: v2
-name: dynatrace-gcp-function
-description: A Helm chart for dynatrace-gcp-function which provides the mechanism to pull all Google Cloud metrics and/or Google Cloud logs into Dynatrace.
+name: dynatrace-gcp-monitor
+description: A Helm chart for dynatrace-gcp-monitor which provides the mechanism to pull all Google Cloud metrics and/or Google Cloud logs into Dynatrace.
 type: application
 version: 0.3.0
```

There is a warning "No newline at end of file"

```suggestion
version: 0.3.0

```